policies/step_annotations.rego

```diff
@@ -0,0 +1,25 @@
+package hacbs.contract.step_annotations
+
+import data.lib
+
+deny[{"msg": msg}] {
+	att := data.attestations[_]
+
+	some step_index
+
+	# null annotations show up when there are none defined
+	# can there be null annotations?
+	att.predicate.buildConfig.steps[step_index].annotations != null
+
+	# if an annotations is defined, it shows up in a list
```

Comment:
I'm surprised that it is a list of strings!

Reply:
Digging through the Chains code, I don't see how `annotations` will ever not be `null` :smile: 
It's defined [here](https://github.com/tektoncd/chains/blob/647e29892e2facd8897286fbc782bed74b2264c6/pkg/chains/formats/intotoite6/buildconfig.go#L37) but I can't find any usage of it. @bcaton85 wdyt?

Reply:
odd, Annotations doesn't seem to be referenced anywhere.